account/manager.go

```diff
@@ -624,9 +702,20 @@ func (m *manager) resumeAccount(ctx context.Context, account *Account, // nolint
 				}
 			}
 
-			acctKey := account.TraderKey.PubKey.SerializeCompressed()
-			contextLabel := fmt.Sprintf(" poold -- "+
-				"AccountCreation(acct_key=%x)", acctKey)
+			txFee, err := m.deriveFeeFromTx(ctx, accountTx)
+			fee := &txFee
+			if err != nil {
+				log.Errorf(
```

Comment:
Ah, all calls to `log.Xxxf()` and `fmt.Errorf()` should use the compact formatting (that way they're easily visually distinguishable from an actual method/function call). See lnd's formatting guide.